pkgs/applications/editors/your-editor/default.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib, gccStdenv, fetchFromGitHub }:
```

Comment:
if this is required, maybe it would be better to override it inside all-packages.nix